snuba/migrations/runner.py

```diff
@@ -76,9 +82,11 @@ def get_status(migration_key: MigrationKey) -> Status:
 
         return migrations
 
-    def run_migration(self, migration_key: MigrationKey) -> None:
+    def run_migration(self, migration_key: MigrationKey, force: bool = False) -> None:
```

Comment:
I noticed the PR that allows to reverse migration introduces another bool field in this public method (same for the run_all). This is a code smell (boolean arguments in public methods, worse if they are multiple - an example: https://medium.com/@amlcurran/clean-code-the-curse-of-a-boolean-parameter-c237a830b7a3). 

Could you have two separate entry points or at least force them to be key-value arguments so they cannot be confused by just passing them in the wrong order ?